tests/dns.rs

```diff
@@ -0,0 +1,20 @@
+use trust_dns_resolver::proto::rr::{RecordType, RData};
+
+#[test]
+fn test_bad_add() {
+    use std::net::*;
+    use trust_dns_resolver::Resolver;
+    use trust_dns_resolver::config::*;
+
+// Construct a new Resolver with default configuration options
+    let resolver = Resolver::new(ResolverConfig::default(), ResolverOpts::default()).unwrap();
+    let ret = resolver.lookup("www.fillbit.io.", RecordType::CNAME).unwrap();
```

Comment:
please use another random name, one of qovery for exmaple

Reply:
Was not supposed to commit it.
But moved the test and added a cname on qovery.io for that